app/assets/javascripts/discourse/app/lib/d-popover.js

```diff
@@ -39,13 +38,9 @@ export function showPopover(event, options = {}) {
     ? event.target._tippy
     : setup(event.target, options);
 
-  // hangs on legacy ember
-  if (!isLegacyEmber) {
-    begin();
```

Comment:
That was accidentally removed 😃

Reply:
Well, now we're ending a runloop without starting it, so that doesn't seem right 😉

Reply:
🤦